kubernetes/loculus/values.yaml

```diff
@@ -26,7 +26,10 @@ dataUseTerms:
     open: https://#TODO-MVP/open
     restricted: https://#TODO-MVP/restricted
 s3:
-  enabled: false
+  enabled: true
+  bucket:
+    endpoint: hel1.your-objectstorage.com
+    bucket: loculus-preview-private
```

Comment:
I changed the schema to require accessKey and secretKey if the feature is enabled. Could that be causing issues?

Reply:
Ah probably also that. But also I think the config wouldn't have read the env vars anyways? Maybe with the latest two commits it will work. I set both of the settings, but also in such a way that the config preprocessor should handle the replacing.